draft-normington-jsonpath-latest.txt

```diff
@@ -464,7 +487,52 @@ Semantics
 
    A negative index "j" selects an element of an array of length "len"
    if and only if "0 <= j + len < len", in which case it selects the
-   same element as selected by the non-negative index "j + len".
+   same element as the non-negative index "j + len".
+
+   An array slice matches any node which is an array and selects any
+   elements of the array indexed by the members of the slice, as defined
```

Comment:
Same comment as above, this is a little difficult for me to follow.

Reply:
Tried a rewording. See what you think.